VAMobile/env/env.sh

```diff
@@ -53,6 +53,7 @@ echo "DEMO_PASSWORD=${DEMO_PASSWORD}" >> .env
 
 # set website URLs
 echo "LINK_URL_SCHEDULE_APPOINTMENTS=https://${WEBSITE_PREFIX}va.gov/health-care/schedule-view-va-appointments" >> .env
+echo "LINK_URL_VA_MEDICAL_RECORDS=https://${WEBSITE_PREFIX}va.gov/my-health/medical-records/" >> .env
```

Comment:
Could this be in constant.env instead of the script generating it?

Reply:
I think so, I am not sure if we Need it to be per environment